doc/shell_long/README.md

```diff
@@ -0,0 +1,635 @@
+# 1. Test Objective  
+This guide is to introduce how to execute shell_long test suite via test tool CTP.  The shell_long test case specification is exactly same as general SHELL test case's. The only difference between them is that shell_long test cases ask a little long time to execute.     
+
+# 2. Test Introduction  
+## 2.1 CTP test tool
+Please refer to [ctp usage for shell](https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/shell/shell_guide.md#2-ctp-introduction)   
+the CTP usage for shell_long regression test is as below   
+```bash
+ctp.sh shell -c ~/CTP/conf/shell_runtime.conf
+```
+for regression test, we will copy `shell_template_for_shell_long.conf` to `shell_runtime.conf`   

```

Comment:
Follow same way to describe in in cci guide.
It's better not to describe things related to regression test. We hope that developer users may get guide from here.